src/server/schema/resolvers/mutations/createLife.ts

```diff
@@ -0,0 +1,23 @@
+import { ObjectId } from 'mongodb';
+import { getDatabaseContext, Life } from '../../../database';
+import { GraphQLMutationResolvers } from '../definitions';
+
+const mutation: GraphQLMutationResolvers['createLife'] = async (root, { life }) => {
+    const { collections } = await getDatabaseContext();
+
+    const document: Life = {
+        _id: new ObjectId(),
+        firstName: life.firstName,
+        lastName: life.lastName,
+        birthday: life.birthday,
+        title: life.title,
+        description: life.description,
+        hobbies: life.hobbies,
```

Comment:
One cool trick you can do here is spread life
```
{
  _id: new ObjectId(),
 ...life
}
```

Reply:
Doing this causes and issue because the title field in `life` is optional but it is required by the `Life` class. My work around was overwriting the title after the spread operator again. Not sure if that is the best way to do it...
```
{
  _id: new ObjectId(),
  ...life,
  title: life.title,
};
```